source_test.yaml

```diff
@@ -68,6 +70,7 @@
   versions_from_repo: False
   rest_api_url: 'https://curl.se/docs/vuln.json'
   type: 2
+  ignore_patterns: ['^(?!CURL-).*$']
```

Comment:
actually, I just noticed this line is duplicated with line 74. It also isn't applicable for REST sources right?

Reply:
Thanks for catching that. Fixed in 8c34b659, and yes, by the looks of things, https://github.com/google/osv.dev/blob/f182e67869f4963d1cfc406b75953edd46eb6338/docker/importer/importer.py#L58 is not currently called from https://github.com/google/osv.dev/blob/f182e67869f4963d1cfc406b75953edd46eb6338/docker/importer/importer.py#L652

I've left the entries for the two REST sources as a statement of intent, and added a comment to make it obvious that it's not currently honoured.